src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf

```diff
@@ -0,0 +1,391 @@
+            </trans-unit>
+            <trans-unit id="88">
+                <source>This value should be positive.</source>
+                <target>Qiymat musbat bo'lishi kerak.</target>
+            </trans-unit>
+            <trans-unit id="89">
+                <source>This value should be either positive or zero.</source>
+                <target>Qiymat musbat yoki 0 ga teng bo'lishi kerak.</target>
+            </trans-unit>
+            <trans-unit id="90">
+                <source>This value should be negative.</source>
+                <target>Qiymat manfiy bo'lishi kerak.</target>
+            </trans-unit>
+            <trans-unit id="91">
+                <source>This value should be either negative or zero.</source>
+                <target>Qiymat manfiy yoki 0 ga teng bo'lishi kerak.</target>
+            </trans-unit>
+            <trans-unit id="92">
+                <source>This value is not a valid timezone.</source>
+                <target>Qiymat to'g'ri vaqt zonasi emas.</target>
+            </trans-unit>
+            <trans-unit id="93">
+                <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
+                <target>Ushbu parol ma'lumotlarning tarqalishi tufayli buzilgan va uni ishlatmaslik kerak. Boshqa paroldan foydalaning.</target>
+            </trans-unit>
+            <trans-unit id="94">
+                <source>This value should be between {{ min }} and {{ max }}.</source>
+                <target>Qiymat {{ min }} va {{ max }} oralig'ida bo'lishi shart.</target>
+            </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target>Qiymat to'g'ri xost nomi emas.</target>
+            </trans-unit>
+            <trans-unit id="96">
+                <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
+                <target>Ushbu to'plamdagi narsalar soni {{ compared_value }} dan ko'p bo'lishi kerak.</target>
+            </trans-unit>
+            <trans-unit id="97">
+                <source>This value should satisfy at least one of the following constraints:</source>
+                <target>Qiymat quyidagi cheklovlardan kamida bittasiga javob berishi kerak:</target>
+            </trans-unit>
+            <trans-unit id="98">
+                <source>Each element of this collection should satisfy its own set of constraints.</source>
+                <target>Ushbu to'plamdagi har bir narsa o'ziga xos cheklovlarni qondirishi kerak.</target>
+            </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid International Securities Identification Number (ISIN).</source>
+                <target>Qiymat Qimmatli qog'ozlarning xalqaro identifikatsiya raqami (ISIN) ga mos emas.</target>
+            </trans-unit>
```

Comment:
There is now an additional one (100). Can you add it as well?

Reply:
@fabpot certainly can! but I did not see from the English translations that there are 100

Reply:
Most translation already exists. You should rebase your 4.4 branch to the current 4.4 branch from symfony/symfony. That would also get you the new translation.

Reply:
@fabpot yes I saw! then you can delete this pull request, and I'll fix it and create a new one!